docs/source/intro/run_docker.rst

```diff
@@ -66,56 +128,108 @@ This image contains a fully pre-configured AiiDA environment which makes it part
+See `developement wiki <https://github.com/aiidateam/aiida-core/wiki/Development-environment>`_ for more detalis.
 
-         $ docker cp test.txt aiida-container:/home/aiida
+.. dropdown:: Copy files from your computer to the container
 
-   .. grid-item-card:: Persist data across different containers
+   .. tab-set::
 
-      If you stop the container (`docker stop` or simply `Ctrl+D` from the container) and start it again, any data you created will persist.
+      .. tab-item:: Docker CLI
 
-      .. code-block:: console
+         Use the ``docker cp`` command if you need to copy files from your computer to the container or vice versa.
 
-         $ docker start -i aiida-container
+         For example, to copy a file named ``test.txt`` from your current working directory to the ``/home/aiida`` path in the container, run:
 
-      However, if you remove the container, **all data will be removed as well**.
+         .. code-block:: console
 
-      .. code-block:: console
+            $ docker cp test.txt aiida-container-demo:/home/aiida
 
-         $ docker stop aiida-container
-         $ docker rm aiida-container
 
-      The preferred way to persistently store data is to `create a volume <https://docs.docker.com/storage/volumes/>`__.
+.. dropdown:: Persist data across different containers
 
-      To create a simple volume, run:
+   The lifetime of the data stored in a container is limited to the lifetime of that particular container.
 
-      .. code-block:: console
+   If you stop the container (`docker stop` or simply `Ctrl+D` from the container) and start it again, any data you created will persist.
+   However, if you remove the container, **all data will be removed as well**.
+
+   .. code-block:: console
+
+      $ docker rm aiida-container-demo
+
+   The preferred way to persistently store data is to `create a volume <https://docs.docker.com/storage/volumes/>`__.
+
+   .. tab-set::
+
+      .. tab-item:: Docker CLI
+
+         To create a simple volume, run:
+
+         .. code-block:: console
+
+            $ docker volume create container-home-data
+
+         In this case, one needs to specifically mount the volume very first time that the container is being created:
+
+         .. parsed-literal::
+
+            $ docker run -it --name aiida-container-demo -v container-home-data:/home/aiida aiidateam/aiida-core:latest bash
+
+         Starting the container with the above command ensures that any data stored in the ``/home/aiida`` path within the container is stored in the ``conatiner-home-data`` volume and therefore persists even if the container is removed.
+
+         When installing packages with pip, use the ``--user`` flag to store the Python packages installed in the mounted volume (if you mount the home specifically to a volume as mentioned above) permanently.
+         The packages will be installed in the ``/home/aiida/.local`` directory, which is mounted on the ``container-home-data`` volume.
+
```

Comment:
I'm getting a bit confused by the terminology. Is the volume mounted on a directory, or the directory mounted on the volume?

Reply:
Volume mounted on a container (pointing to a specific path in container) = specific container path is really a link back to the volume, so any data written to it is really written to the volume. Destroy the container and lunch a new one with the same volume mounted, and voila, you're back to work!